src/flows/publisher_data/update_pocket_score_data_flow.py

```diff
@@ -101,95 +103,98 @@
+
+            if save_cnt > 0 or open_cnt > 0 or favorite_cnt > 0 or share_cnt > 0:
+                open_uv = float(open_cnt) + 2.2 + save_cnt * 0.05
+                open_dv = (float(save_cnt) - open_cnt) + 7.8
+                open_dv = open_dv if save_cnt >= open_cnt else (open_dv + open_uv) * 4
+                fav_uv = float(favorite_cnt) + 0.2 + open_cnt * 0.02
+                fav_dv = (float(open_cnt) - favorite_cnt) + 9.8
+                fav_dv = fav_dv if open_cnt >= favorite_cnt else (fav_dv + fav_uv) * 4
+                share_uv = float(share_cnt) + 0.2 + open_cnt * 0.02
+                share_dv = (float(open_cnt) - share_cnt) + 9.8
+                share_dv = share_dv if open_cnt >= share_cnt else (share_dv + share_uv) * 4
+
+                open_score = scipy.special.betaincinv(open_uv, open_dv, inc_beta_var)
+                fav_score = scipy.special.betaincinv(fav_uv, fav_dv, inc_beta_var)
+                share_score = scipy.special.betaincinv(share_uv, share_dv, inc_beta_var)
+
+                total_score = (open_score * open_var + fav_score * fav_var + share_score * share_var) / (
+                        open_var + fav_var + share_var)
+
+                result.append({'save_cnt': save_cnt, 'open_uv': open_uv, 'open_dv': open_dv, 'fav_uv': fav_uv,
+                               'fav_dv': fav_dv, 'share_uv': share_uv, 'share_dv': share_dv,
+                               'open_score': open_score, 'favorite_score': fav_score, 'share_score': share_score,
+                               'total_score': total_score, 'publisher_id': publisher_id, 'content_id': content_id})
+    return result
+
+
+def insert_args(rows: [dict[str, Union[float, int]]]) -> [Tuple[int, int]]:
+    return [(
+        x['publisher_id'],
+        x['content_id']
+    ) for x in rows]
+
+
+def update_args(rows: [dict[str, Union[float, int]]]) -> [Tuple]:
+    return [(
+        x['save_cnt'],
+        x['open_uv'],
+        x['open_dv'],
+        x['fav_uv'],
+        x['fav_dv'],
+        x['share_uv'],
+        x['share_dv'],
+        x['open_score'],
+        x['favorite_score'],
+        x['share_score'],
+        x['total_score'],
+        x['publisher_id'],
+        x['content_id']
+    ) for x in rows]
+
+
+@task(timeout=10 * 60)
+def load(rows: [dict[str, Union[float, int]]]):
+    # TODO: Handle values less than max_content_id
+    execute_many = PocketMySQLExecuteMany(MYSQL_PUBLISHER_CONNECTION_DICT)
+
+    one_dict = [x for x in rows if x['content_id'] <= max_content_id]
+    two_dict = [x for x in rows if x['content_id'] > max_content_id]
+
+    execute_many(query=INSERT_TO_CONTENT_V1_SQL, args=insert_args(one_dict))
+    execute_many(query=INSERT_TO_CONTENT_V2_SQL, args=insert_args(two_dict))
+
+    execute_many(query=UPDATE_CONTENT_V1_SQL, args=update_args(one_dict))
+    execute_many(query=UPDATE_CONTENT_V2_SQL, args=update_args(two_dict))
 
 
 with Flow(FLOW_NAME, schedule=get_interval_schedule(minutes=60)) as flow:
```

Comment:
By default tasks run in a single thread. To leverage multi-threading with 16 threads:
```suggestion
with Flow(FLOW_NAME, schedule=get_interval_schedule(minutes=60), executor=LocalDaskExecutor(num_workers=16)) as flow:
```
`from prefect.executors import LocalDaskExecutor` at the top.